src/main/blackbox/blackbox.h

```diff
@@ -43,6 +43,7 @@ typedef enum FlightLogEvent {
     FLIGHT_LOG_EVENT_INFLIGHT_ADJUSTMENT = 13,
     FLIGHT_LOG_EVENT_LOGGING_RESUME = 14,
     FLIGHT_LOG_EVENT_FLIGHTMODE = 30, // Add new event type for flight mode status.
+    FLIGHT_LOG_EVENT_DISARM = 31,
```

Comment:
Any idea why this enum is sparse? We could probably use one of the unused values below 30 for this.

Reply:
I have no idea, I looked around but couldn't see why, probably requires some digging into git history to figure it out.  @thenickdude might know?

Reply:
Chances are very good that a fork decided to add a healthy margin to the last known ID so they didn't collide with IDs that the upstream added later. There's no technical reason it needs to be spare.

Reply:
@thenickdude thanks for the insight, I'll just use the lowest available number that's not been used before.